script.js

```diff
@@ -0,0 +1,35 @@
+const hamburger = document.querySelector('.ham');
+const navSub = document.querySelector('.nav-sub');
+const title = document.querySelector('.title');
+
+function openCloseMenu() {
+  hamburger.classList.toggle('change');
+  navSub.classList.toggle('nav-change');
+  title.classList.toggle('title-change');
+}
+
+hamburger.addEventListener('click', openCloseMenu);
```

Comment:
- [ ] **Optional:** When the menu is open, you could restrict scroll and/or dim the main page.

Reply:
@brytebee  Thank you so much, I fixed that issue please let me know if I can do more things to improve the website.
## I really appreciate your optional comments :+1: